### PR TITLE
New Entry: C Operators

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

From target keyword list.
### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c/concepts/operators/operators.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  - 'Operators'
+  - 'Arithmetic'
+  - 'Comparison'
+  - 'Logical'
+CatalogContent:
+  - 'learn-c'
+  - 'paths/computer-science'
+---
+
+Operators are used to perform operations on variables and values. They are symbols that tell the compiler to perform specific mathematical or logical functions. The C language provides the following types of operators:
```
Comment (Dusch4593) [inline content/c/concepts/operators/operators.md:17]:
```suggestion
**Operators** are used to perform operations on variables and values. They are symbols that tell the compiler to perform specific mathematical or logical functions. The C language provides the following types of operators:
```

File: content/c/concepts/operators/operators.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+- Bitwise Operators
+- Assignment Operators
+- Misc Operators
+
+## Arithmetic Operators
+
+C has the following basic arithmetic operators to perform common mathematical operations:
+
+| Operator | Name           | Description                                                 |
+| -------- | -------------- | ----------------------------------------------------------- |
```
Comment (Dusch4593) [inline content/c/concepts/operators/operators.md:31]:
```suggestion
| :--------: | :--------------: | ----------------------------------------------------------- |
```

File: content/c/concepts/operators/operators.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+x++;       // x is now 3
+x--;       // x is now 2
+```
+
+## Relational Operators
+
+Relational operators compare two values and return `true` or `false`.
+
+| Operator | Name                     | Description                                                |
+| -------- | ------------------------ | ---------------------------------------------------------- |
```
Comment (Dusch4593) [inline content/c/concepts/operators/operators.md:56]:
```suggestion
| :--------: | :------------------------: | ---------------------------------------------------------- |
```

File: content/c/concepts/operators/operators.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+  // This code won't run.
+}
+```
+
+## Logical Operators
+
+Logical operators operate on boolean values and return boolean values.
+
+| Operator | Name | Description                                                        |
+| -------- | ---- | ------------------------------------------------------------------ |
```
Comment (Dusch4593) [inline content/c/concepts/operators/operators.md:82]:
```suggestion
| :--------: | :----: | ------------------------------------------------------------------ |
```

File: content/c/concepts/operators/operators.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+  // This code won't run.
+}
+```
+
+## Bitwise Operators
+
+Bitwise operators manipulate individual bits in an integer.
+
+| Operator | Name        | Description                                                    |
+| -------- | ----------- | -------------------------------------------------------------- |
```
Comment (Dusch4593) [inline content/c/concepts/operators/operators.md:105]:
```suggestion
| :--------: | :-----------: | -------------------------------------------------------------- |
```

File: content/c/concepts/operators/operators.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+x = 19 >> 2;
+// x is 4 : 10011 >> 2 = 00100
+```
+
+## Assignment Operators
+
+Assignment operators assign a value to the right to a variable on the left.
+
+| Operator | Name                      | Description                                |
+| -------- | ------------------------- | ------------------------------------------ |
```
Comment (Dusch4593) [inline content/c/concepts/operators/operators.md:137]:
```suggestion
| :--------: | :-------------------------: | ------------------------------------------ |
```

File: content/c/concepts/operators/operators.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+| `^=`     | Xor Assignment            | `a ^= 3` is short for `a = a ^ 3`.         |
+| `<<=`    | Left Shift Assignment     | `a <<= 3` is short for `a = a << 3`.       |
+| `>>=`    | Right Shift Assignment    | `a >>= 3` is short for `a = a >> 3`.       |
+
+## Misc Operators
+
+Misc operators don't fit into a particular category.
+
+| Operator   | Name                   | Description                                                                     |
+| ---------- | ---------------------- | ------------------------------------------------------------------------------- |
```
Comment (Dusch4593) [inline content/c/concepts/operators/operators.md:155]:
```suggestion
| :----------: | :----------------------: | ------------------------------------------------------------------------------- |
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Finished with my review and left some comments/suggestions.

**Comment by SSwiniarski** [comment]
@Dusch4593, I applied your suggestions.

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski LGTM! 👍🏻

**Review by KTom101 — COMMENTED** [review]
@SSwiniarski Looks good. I only have a couple suggestions.

**Review by KTom101 — APPROVED** [review]

**Pull request merged**